lib/ansible/modules/source_control/github_issue.py

```diff
@@ -34,13 +38,12 @@
     description:
         - Get various details about issue depending upon action specified.
     default: 'get_status'
-    choices:
-        - ['get_status']
+    choices: ['get_status']
```

Comment:
Use a yaml list instead of a python list.

Reply:
ok.